services/audit-service/pom.xml

```diff
@@ -56,6 +125,13 @@
         </dependency>
     </dependencies>
     <repositories>
+        <repository>
```

Comment:
I hate having 3rd party repos in here. Do we really need to depend on the Cloudera Hadoop version for what we're doing, or would it make more sense to depend on stock? Could see an argument both ways...

Reply:
I don't have a dog in this fight.  I was just trying to carry forward the precedent set in the warehouse pom.